sormas-backend/src/main/java/de/symeda/sormas/backend/disease/DiseaseConfiguration.java

```diff
@@ -32,7 +32,12 @@
 
 	public static DiseaseConfiguration build(Disease disease) {
 		DiseaseConfiguration configuration = new DiseaseConfiguration();
+
 		configuration.setDisease(disease);
+		configuration.setExtendedClassification(disease.isDefaultExtendedClassification());
```

Comment:
This should be removed - the system should be able to deal with null values because we have the defaults

Reply:
you are correct, in earlier versions this was mandatory but now with properties it works :)